prow/cmd/deck/tide_test.go

```diff
@@ -130,13 +154,20 @@ func TestFilterHidden(t *testing.T) {
 			log:         logrus.WithField("agent", "tide"),
 		}
 
-		gotQueries, gotPools := ta.filterHidden(test.queries, test.pools)
+		gotQueries := ta.filterHiddenQueries(test.queries)
+		gotPools := ta.filterHiddenPools(test.pools)
+		gotHist := ta.filterHiddenHistory(test.hist)
 		if !equality.Semantic.DeepEqual(gotQueries, test.expectedQueries) {
 			t.Errorf("expected queries:\n%v\ngot queries:\n%v\n", test.expectedQueries, gotQueries)
 		}
 		if !equality.Semantic.DeepEqual(gotPools, test.expectedPools) {
 			t.Errorf("expected pools:\n%v\ngot pools:\n%v\n", test.expectedPools, gotPools)
 		}
+		// equality.Semantic.DeepEqual doesn't like the unexported fields in time.Time.
```

Comment:
We're not dealing with k8s objects anyway so we don't _need_ semantic equality here